WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -45,10 +46,17 @@
 
     <DockPanel>
       <TextBlock Text="Network anonymization (Tor)" />
-      <ToggleSwitch IsChecked="{Binding Settings.UseTor}" />
+      <ComboBox HorizontalAlignment="Stretch" ItemsSource="{Binding TorModes}" SelectedItem="{Binding Settings.UseTor}" Width="250">
```

Comment:
@kiminuo The selection here doesn't even seem to be working. When I try to change it, it doesn't pop the restart message, and after restarting the software it resets back.

Reply:
for me the pop up does show and restarting the software keeps the right selection

Reply:
Something is clearly wrong, at least on Windows.

Original value: `Disabled`
`Disabled` -> `Enabled`. No restart message (there should be)
`Enabled` -> `Enabled connect only` restart message pops
`Enabled connect only` -> `Disabled` restart message stays (since Disabled is the original it should have been hidden)
`Disabled` -> `Enabled` no restart message (there should be)

https://github.com/zkSNACKs/WalletWasabi/assets/16364053/7fb7733b-7238-4e0a-b6da-1af474c6f7fa

Reply:
that also happened to me a few days ago.
my comment https://github.com/zkSNACKs/WalletWasabi/pull/12991#discussion_r1609589133, was a quick try just now, setting it from enabled to disabled which worked ok